ktoml-core/src/iosSimulatorArm64Main/kotlin/com/akuleshov7/ktoml/utils/UtilsSimulator.kt

```diff
@@ -0,0 +1,15 @@
+/**
+ * Specific implementation for utilities
+ */
+
+package com.akuleshov7.ktoml.utils
+
+@Suppress("MAGIC_NUMBER")
+internal actual fun StringBuilder.appendCodePointCompat(codePoint: Int): StringBuilder = when (codePoint) {
```

Comment:
does it relate to this PR?

Reply:
Yeah, unfortunately. I need to implement these ‘actual’ utils for simulator also :(